xwiki-pro-macros-ui/src/main/resources/XWiki/Macros/Panel.xml

```diff
@@ -540,7 +564,16 @@ This macro supports all Atlassian Confluence parameters as of version 7.9.</cont
         $!xcontext.macro.content
       #end
     )))
-  )))
+    #if( $panelFooter != '')
+      (% class="macro-panel-footer" style="$panelFooterStyle"%)(((
```

Comment:
You should escape the panelFooterStyle in the current syntax, since it contains data from the users.